teams/Ballerina.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+org= "ballerinax"
+name= "microsoft.teams"
+version= "0.1.0"
```

Comment:
In the master branch, connector version should be in SNAPSHOT

Reply:
Fixed thanks